### DISCUSSION TITLE
Question: do you plan to provide installer file in the future?

### DISCUSSION THREAD

**Discussion by bartfer** [Difegue/Stylophone#42] [Q&A] [answered]
Hi,

Seems like a pretty good project. My issue is that I can't setup proper build tools easily because of special limitations.
Do you plan to provide an installer file with future releases?

Another question is do you intend this project to be free and opensource in the future as well? (I've read the 2 articles on your site which gave me the impression you don't intend to monetize this other than the microsoft store. But would be interested in your plans as I'm currently searching for the proper longterm MPDClient on Windows preferably not webbased :) )

(also installation/building readme would be nice :) )

Thanks for making this project!

**Comment by Difegue** [comment, marked as answer]
My monetization scheme for Stylophone is to provide compiled releases only through the MS Store. (And it's been working pretty well so far!)   

It's a bit of a "if you really want to get it for free, it's open-source so build it yourself" thing. (and a few people do exactly that, according to github analytics 🙂) 
I don't plan on closing up the source or anything, so compiling yourself will always give you the same experience as the MS Store version.

Since the app is just UWP, there's nothing special involved in building it -- Just Visual Studio with the required workload -- So I don't think it's necessary to add detailed build instructions.  
(The iOS port requires Xamarin.iOS and a Mac, but you can just skip it 😁)

**Comment by bartfer** [comment]
I understand, thanks for the quick response and nice approach! :)
